README.md

```diff
@@ -126,6 +126,49 @@ Try the following examples to learn more about Dapr's Java SDK:
 
 Please, refer to our [Javadoc](https://dapr.github.io/java-sdk/) website.
 
+#### Reactor API
+
+The Java SDK for Dapr is built using [Project Reactor](https://projectreactor.io/). It provides an asynchronous API for Java. When consuming a result is consumed synchronously, as in the examples above, the `block()` method is used.
+
+The code below does not make any API call, it simply returns the [Mono](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html) publisher object:
```

Comment:
> The code below does not make any API call
It's better to phrase that it doesn't start execution or "nothing happens" in the wording of the reactor docs.

Reply:
done.